Caddyfile.dev

```diff
@@ -3,34 +3,54 @@
     auto_https off
 }
 
-:8080 {
-    file_server
-    try_files {path} index.html
+# Bind to port 8080
+:8080
 
-    header Access-Control-Allow-Methods "GET, PUT, POST, DELETE, OPTIONS"
-    header Access-Control-Allow-Origin "*"
-    header Content-Security-Policy "
-      default-src 'self';
-      connect-src 'self'
-        api.bitclout.com bitclout.com:*
-        api.bitpop.dev
-        bitclout.me:* api.bitclout.me:*
```

Comment:
Thanks for this fix. It appears this removes bitclout.me and api.bitclout.me, which will break local dev, but I just pushed a minimal fix.

Reply:
ah of course :) how did i miss that.